ert/data/record/_transmitter.py

```diff
@@ -78,13 +96,50 @@ async def _load_numerical_record(self) -> NumericalRecord:
     async def _load_blob_record(self) -> BlobRecord:
         raise NotImplementedError("not implemented")
 
+    @abstractmethod
+    async def get_recordtree_transmitters(
+        self,
+        trans_records: Dict[str, str],
+        record_type: RecordType,
+        _path: Optional[str] = None,
+    ) -> Dict[str, "RecordTransmitter"]:
+        raise NotImplementedError("not implemented")
+
+    async def load_numerical_recordtree(
+        self,
+        transmitters: Dict[str, "RecordTransmitter"],
+    ) -> NumericalRecordTree:
+        flat_record_dict = {
+            rec_path: await transmitter._load_numerical_record()
+            for rec_path, transmitter in transmitters.items()
+        }
+        record_dict = _unflatten_record_dict(flat_record_dict)
+        return NumericalRecordTree(record_dict=record_dict)
+
+    async def load_blob_recordtree(
+        self,
+        transmitters: Dict[str, "RecordTransmitter"],
+    ) -> BlobRecordTree:
+        record_path_dict = {
+            rec_path: await transmitter._load_blob_record()
```

Comment:
Another private method.

Reply:
Yes, but it is called from the `RecordTransmitter` class, where it is fine to call private.